### PR TITLE
Linked Madrid

### PR DESCRIPTION
**Pull request by Esteve2604**
We are a group of university students and we would like to use that domain to make our knowledge graph available. Thank you

### CODE REVIEW

**Comment by dgarijo** [comment]
@Esteve2604 can you please add your username in the readme too?
Also, it looks like you are redirecting to a namespace you don't own (https://example.com/ ). Can you please point to where you plan on doing the redirects?

**Comment by dgarijo** [comment]
ping @Esteve2604

File: linkedMadrid/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Linked-Madrid is a dataset that aims at providing data from different datasets published by Madrid Government, from energy to educational centers, to make it available for everyone. This datasets will also be linked with big knowledge graphs such as Wikidata or DBpedia.
+
+## Contacts
+
+- estebanasperuiz@gmail.com
+
+
+## Reference
+
+Esteban Aspe
```
Comment (TallTed) [inline linkedMadrid/README.md:12]:
```suggestion
Esteban Aspe Ruiz
* estebanasperuiz@gmail.com
* GitHub username: Esteve2604
```

**Comment by dgarijo** [comment]
Closing due to inactivity. Please reopen if appropriate

**Pull request closed without merging**